zipkin-storage/mysql-v1/src/main/java/zipkin2/storage/mysql/v1/MySQLSpanConsumer.java

```diff
@@ -86,30 +86,30 @@ public Void apply(DSLContext create) {
         Map<TableField<Record, ?>, Object> updateFields = new LinkedHashMap<>();
         if (timestamp != 0L) {
           // tentatively we can use even a shared timestamp
-          insertSpan.set(ZIPKIN_SPANS.START_TS, timestamp);
+          insertSpan = insertSpan.set(ZIPKIN_SPANS.START_TS, timestamp);
```

Comment:
note: changes like this weren't bugs, just the new error-prone plugin is strict about builders and we can't easily affect the jooq code to add the annotation to it. doing this is harmless, except the cruft